### ISSUE TITLE
Problems about pointer-events : none

### ISSUE THREAD

**Issue by phu130201** [radix-ui/primitives#3298]
(https://stackoverflow.com/questions/79295806/pointerevent-none-happened-when-close-modal-in-nextjs15)
I have this issues and I can not fix it.

**Comment by MaciejGarncarski** [comment]
Hi, I've had the same problem, and I fixed it by updating `@radix-ui/react-dialog`, `@radix-ui/react-popover` and `@radix-ui/react-alert-dialog` to version `1.1.4`

**Comment by phu130201** [comment]
> Hi, I've had the same problem, and I fixed it by updating `@radix-ui/react-dialog`, `@radix-ui/react-popover` and `@radix-ui/react-alert-dialog` to version `1.1.4`

<img width="458" alt="image" src="https://github.com/user-attachments/assets/9ef62a94-95d8-4d1f-baa5-88066f746a28" />
I already changed the react-dialog to version `1.1.4` but still errors.

**Comment by Skayfa** [comment]
I have the same issue :/

**Comment by Lexachoc** [comment]
To avoid this, the following anatomy should fix this issue:

```jsx
<DropdownMenu>
    <DropdownMenuTrigger>
    </DropdownMenuTrigger>
    <DropdownMenuContent>
        <DropdownMenuLabel>
        </DropdownMenuLabel>
        <DropdownMenuSeparator/>
        <DropdownMenuGroup>
            <Dialog>
                <DialogTrigger asChild>
                    <DropdownMenuItem>
                    </DropdownMenuItem>
                </DialogTrigger>
                <DialogContent>
                    <DialogHeader>
                        <DialogTitle>
                        </DialogTitle>
                    </DialogHeader>
                </DialogContent>
            </Dialog>
        </DropdownMenuGroup>
    </DropdownMenuContent>
</DropdownMenu>
```

And close the Dialog and then close the DropdownMenu.

The key point here is to wrap the `Dialog` inside the `DropdownMenu` and use `DropdownMenuItem` as the `DialogTrigger` 

I use the above anatomy without problems.

**Comment by phu130201** [comment]
![image](https://github.com/user-attachments/assets/84ec5505-89c3-4080-abba-36697b0cacc6)
I created a modal and wrap it in dialog then the editor field can not scroll or hover. Any solutions guys ? ( it prevent me from pointer events and scroll )

**Comment by Lexachoc** [comment]
> ![image](https://private-user-images.githubusercontent.com/71860908/400106728-84ec5505-89c3-4080-abba-36697b0cacc6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU5NTYxMTEsIm5iZiI6MTczNTk1NTgxMSwicGF0aCI6Ii83MTg2MDkwOC80MDAxMDY3MjgtODRlYzU1MDUtODljMy00MDgwLWFiYmEtMzY2OTdiMGNhY2M2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA0VDAxNTY1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3OWI5NzllOTQzOGM3YTFiYTMwOGY4ZjUyODRhM2ViYjk1MGM1YTY1NDAyYTE1M2MzNWM1MDM2ZmMyZDAzYjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.CyNuGhUwPGLBIi_L_ZElZhOR1iB1NEplgLuyh9oGXIA) I created a modal and wrap it in dialog then the editor field can not scroll or hover. Any solutions guys ? ( it prevent me from pointer events and scroll )
> 
> ```
> <Dialog>
>           <DialogTrigger asChild>{trigger}</DialogTrigger>
>           <DialogContent className="bg-[#1C1C1E] !max-w-2xl border-themeGray ">
>             {children}
>           </DialogContent>
>         </Dialog>
> 
> 
> <SimpleModal
>         trigger={
>           <span>
>             <Card className="border-themeGray cursor-pointer first-letter:rounded-2xl overflow-hidden">
>               <CardContent className="p-3 bg-[#1A1A1D] flex gap-x-6 items-center ">
>                 <Avatar className="cursor-pointer">
>                   <AvatarImage src={userImage} alt="user" />
>                   <AvatarFallback>U</AvatarFallback>
>                 </Avatar>
>                 <CardDescription className="text-themeTextGray">
>                   Type / to add elements to your post...
>                 </CardDescription>
>               </CardContent>
>             </Card>
>           </span>
>         }
>       >
>         <div className="flex gap-x-3">
>           <Avatar className="cursor-pointer">
>             <AvatarImage src={userImage} alt="user" />
>             <AvatarFallback>U</AvatarFallback>
>           </Avatar>
>           <div className="flex flex-col">
>             <p className="text-themeTextGray text-sm capitalize">{username}</p>
>             <p className="text-sm captialize text-themeTextGray">
>               Posting in{" "}
>               <span className="font-bold capitalize text-themeTextWhite">
>                 {name}
>               </span>
>             </p>
>           </div>
>         </div>
>         <PostContent channelid={channelid} />
>       </SimpleModal>
> ```

Is this the same issue or a different one? Did you manage to fix your original question?

For this one, try adding:
```
onWheel={(e) => e.stopPropagation()}
onTouchMove={(e) => e.stopPropagation()}
```
to the `DialogContent`.

If you haven't addressed the original problem, this approach may not work.

**Comment by phu130201** [comment]
Yeah after fews research, finally I fixed it by wrap the children inside Overlay inside the DialogContent.
```
  <Dialog>
          <DialogTrigger asChild>{trigger}</DialogTrigger>
          <DialogContent className="bg-[#1C1C1E] !max-w-2xl border-themeGray">
            <Overlay>{children}</Overlay>
          </DialogContent>
        </Dialog>
```
Hope you guys fix this issue.

**Comment by phu130201** [comment]
Yeah after fews research, finally I fixed it by wrap the children inside Overlay inside the DialogContent.
```
  <Dialog>
          <DialogTrigger asChild>{trigger}</DialogTrigger>
          <DialogContent className="bg-[#1C1C1E] !max-w-2xl border-themeGray">
            <Overlay>{children}</Overlay>
          </DialogContent>
        </Dialog>
```
Hope you guys fix this issue.

**Comment by FredrikMWold** [comment]
Anybody figure something out here? I have the same issue, but only if I update from 1.1.2 to 1.1.4

**Comment by johot** [comment]
We had this problem today, we solved it by updating the react-popover package from 1.1.1 to 1.1.4 (we believe the 1.1.2 package also worked), we also update the react-dialog package but not sure if that made any difference.

**Comment by abartolo** [comment]
@FredrikMWold Thanks, I made sure all those libraries that have backdrop are using the same version and it worked for me.

**Comment by xyeres** [comment]
Update your radix packages, solved this demon issue for me as well. Spent too much time on this one xD
